lib/usd/translators/lightWriter.cpp

```diff
@@ -222,4 +223,114 @@ void PxrUsdTranslators_AreaLightWriter::Write(const UsdTimeCode& usdTime)
         usdTime, primSchema, lightFn, _GetSparseValueWriter());
 }
 
+#ifdef UFE_VOLUME_LIGHTS_SUPPORT
+PXRUSDMAYA_REGISTER_WRITER(volumeLight, PxrUsdTranslators_VolumeLightWriter);
+
+PxrUsdTranslators_VolumeLightWriter::PxrUsdTranslators_VolumeLightWriter(
+    const MFnDependencyNode& depNodeFn,
+    const SdfPath&           usdPath,
+    UsdMayaWriteJobContext&  jobCtx)
+    : UsdMayaPrimWriter(depNodeFn, usdPath, jobCtx)
+{
+    MPlug   plug = depNodeFn.findPlug("lightShape");
+    VtValue lightShape
+        = UsdMayaWriteUtil::GetVtValue(plug, MayaUsd::Converter::getUsdTypeName(plug), false);
+    if (lightShape.IsEmpty()) {
+        return;
+    }
+
+    if (lightShape == 1) {
```

Comment:
Note sure if we use an enum than the number 1 is better. It is an arbitrary number either way...